packages/shared/web/src/dashboard-view/hooks/use-liquidity-list.tsx

```diff
@@ -189,78 +180,70 @@ export const useLiquidityList = (
       const walletBalance = account
         ? account.balances.find((t) => t.tokenId === y.underlying.id)
         : undefined;
-      const organicApy = (y.organicAPY || 0) + (y.feeAPY || 0);
+      const organicApy = (y.apy.organicAPY || 0) + (y.apy.feeAPY || 0);
```

Comment:
Let's change the YieldViews.ts file to do this math upstream and remove it here. I'd like to remove as much logic like this as posssible from the components.